business/labels_builder.go

```diff
@@ -0,0 +1,142 @@
+package business
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	destination                = "destination"
+	source                     = "source"
+	regexGrpcResponseStatusErr = "^[1-9]$|^1[0-6]$"
+	regexResponseCodeErr       = "^0$|^[4-5]\\\\d\\\\d$"
+)
+
+type LabelsBuilder struct {
+	side     string
+	peerSide string
+	protocol string
+	labelsKV []string
+}
+
+func NewLabelsBuilder(direction string) *LabelsBuilder {
```

Comment:
I'm not sure what is this.
In the business package is better to give more meaninful naming like "metrics_labels_builder", otherwise it sounds like an aux/helper class.

Note that there are other "helper" similar methods for k8s label management, that's why a "labels_builder" looks odd to me.

Reply:
Right, I'll disambiguate by adding "Metrics" in the name here